xbmc/guilib/GUIBaseContainer.h

```diff
@@ -152,8 +154,8 @@ class CGUIBaseContainer : public IGUIContainer
 
   int m_pageControl;
 
-  std::vector<CGUIListItemLayout> m_layouts;
-  std::vector<CGUIListItemLayout> m_focusedLayouts;
+  std::list<CGUIListItemLayout> m_layouts;
+  std::list<CGUIListItemLayout> m_focusedLayouts;
```

Comment:
My commit message explains why.
Your review doesn't explain anything. You say it might not be, but that's just generic blah blah. That's not helpful.